compiler/fir/tree/src/org/jetbrains/kotlin/fir/types/FirTypeUtils.kt

```diff
@@ -138,3 +138,19 @@ private fun ConeTypeParameterType.hasNotNullUpperBound(): Boolean {
         }
     }
 }
+
+fun FirTypeRef.containsNull(): Boolean =
```

Comment:
The name is a bit strange. `canBeNull`?

Reply:
I followed this one: https://github.com/JetBrains/kotlin/blob/master/compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/KotlinUtils.kt#L36, but yeah, sounds strange.

Reply:
Renamed.